xml/System.IO/FileSystemInfo.xml

```diff
@@ -1289,8 +1289,8 @@ On Unix platforms that do not support creation or birth time, this property retu
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns the original path. Use FullName or Name properties for the full path or file/directory name.</summary>
+        <returns>A <see cref="System.String" /> with the original path.</returns>
```

Comment:
```suggestion
        <returns>A string with the original path.</returns>
```
A link to the String class appears immediately before this text, so the cref here is redundant.